src/pages/console/uibuilder/textfieldtoarea.mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Convert TextField to TextAreaField in Figma`,
+  description: `How to convert existing TextField primitive components to TextAreaField primitive components`
+};
+
+After [upgrading](/console/uibuilder/figmatocode/#upgrading-figma-files) to the current Figma community file. You can update any existing TextFields into TextAreaFields by selecting the desired component and changing the instance of the TextField component to the TextAreaField component.
```

Comment:
Copy change suggestion --

After upgrading to the current Figma community file, you can change any existing TextFields into TextAreaFields by selecting the desired TextField and choosing TextAreaFields under the Swap Instance dropdown.

Reply:
Thanks, updated.